source/examples/generated/kmm/Greeting.codeblock.quick-start.kt

```diff
@@ -1,16 +1,16 @@
-val config = RealmConfiguration(schema = setOf(Task::class))
-val realm = Realm(config)
+val config = RealmConfiguration.with(schema = setOf(Task::class))
+val realm = Realm.open(config)
 realm.writeBlocking {
     this.copyToRealm(Task().apply {
```

Comment:
nitpick :) you can ommit this
```suggestion
    copyToRealm(Task().apply {
```